README.md

```diff
@@ -6,22 +13,19 @@ That is, it will read in following order.
 
 * Read `appsettings.json`
 
-## Supported platforms
-
-* By default, the solution is built with dependencies on Secp256k1.Net and NSec (which have native dependencies that need to be built for every platform, and we right now only provide Linux binaries). This version is published as `DotNetLightning.Core` on Nuget.
-* For a more portable version (but possibly less performant) you can compile with BouncyCastle support, passing /p:BouncyCastle=True to your build. This version is published as `DotNetLightning` on Nuget.
+## Supported platforms for the package `DotNetLightning.Core`
 
-## TODO
+The package `DotNetLightning.Core` is published to nuget as two different style. 
```

Comment:
typo: style -> styles
(BTW I would not use the name "DotNetLightning.Core" in this sentence, as nuget names are mentioned below